modules/signing_scriptworker/templates/dep-passwords-mobile.json.erb

```diff
@@ -1,11 +1,11 @@
 {
     "project:mobile:focus:releng:signing:cert:dep-signing": [
-        ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_focus_dep_username"]) %>", "<%= scope.function_secret(["autograph_focus_dep_password"]) %>", ["autograph_focus"], "autograph"]
+        ["https://autograph-external.prod.autograph.services.mozaws.net", "signingscript_focus_dep", "<%= scope.function_secret(["autograph_focus_dep_password"]) %>", ["autograph_focus"], "autograph"]
```

Comment:
I'm pulling the usernames out of `hiera` because they're not secrets - what do you think?